pkg/webui/console/containers/side-bar/navigation/app-side-navigation.js

```diff
@@ -58,7 +58,11 @@ const AppSideNavigation = () => {
   const mqttDisabled = useSelector(selectMqttProviderDisabled)
   const { isMinimized } = useContext(SidebarContext)
 
-  const entityId = app ? app.name ?? appId : appId
+  if (!app) {
+    return null
+  }
```

Comment:
Before this, there were proptype warnings in the console for initial renders, when the app was not yet available.